pkg/bootstrap/testdata/default_golden.json

```diff
@@ -44,6 +44,36 @@
     }
   },
   "static_resources": {
+    "listeners":[{
```

Comment:
So you effectively added an http proxy for the entire mesh ? 

I don't mind it - it is on my TODO list to add back the real http proxy. However it should be named and documented as such, it's certainly not specific to the agent.